cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -62,24 +71,39 @@ The ELB service-linked role is created for you automatically. You can click any
 
 . Click the orange *Continue to Red{nbsp}Hat* button to proceed with account linking:
 +
+.Continue to Red{nbsp}Hat
++
 image::rosa-continue-rh-6.png[]
 
 . If you are not already logged in to your Red{nbsp}Hat account in your current browser's session, you will be asked to log in to your account:
 +
+[NOTE]
+====
+Your AWS account must linked to a single Red{nbsp}Hat organization. 
```

Comment:
Your AWS account must be linked to a Red{nbsp}Hat account before you can create a ROSA cluster.